### PR TITLE
[TT-9199] Revert "[CI] Sync automation: Syncing commits from master"

### PR DESCRIPTION
**Pull request by titpetric**
## Description

This PR reverts commit  https://github.com/TykTechnologies/tyk/commit/0e27df86ae5f9808158b4a875fd03b357b7bef48

## Related Issue

https://tyktech.atlassian.net/browse/TT-9199

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5168/merge
Commit:  
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5310448481)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5168/merge
Commit: d70d273b66f5c4a150feef96f7973fc88dba0a21 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5310775618)

File: .github/workflows/api-tests.yml [hunk 1]
```diff
@@ -127,7 +129,6 @@ jobs:
         TYK_TEST_REDIS: "tyk-redis"
         TYK_TEST_DB_ADMIN: "12345"
         TYK_TEST_GW_SECRET: "352d20ee67be67f6340b4c0605b044b7"
-        TYK_TEST_FEDERATION_HOST: federation
```
[the remark this answers is not in the document]
Comment (titpetric) [reply .github/workflows/api-tests.yml]: Cherry-picked :+1:

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5168/merge
Commit: 5bac5df4e60a1385bd188e9b2b33251ac4322c62 
Triggered by: pull_request (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5311394871)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/release-4.0.14
Commit: 24f68ceae3ec867e502f7bbf5bac8705b890f483 [TT-9199] Revert "[CI] Sync automation: Syncing commits from master" (#5168)



## Description


This PR reverts commit
https://github.com/TykTechnologies/tyk/commit/0e27df86ae5f9808158b4a875fd03b357b7bef48

## Related Issue





https://tyktech.atlassian.net/browse/TT-9199

## Motivation and Context



## How This Has Been Tested






## Screenshots (if appropriate)

## Types of changes



- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test
coverage to functionality)

## Checklist





- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning
why it's required
- [ ] I would like a code coverage CI quality gate exception and have
explained why

---------

Co-authored-by: Tit Petric <tit@tyk.io>
Co-authored-by: Ilija Bojanovic <ilijabojanovic@gmail.com>
Triggered by: push (@titpetric)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5311701988)

**Pull request merged**